applications/ConvectionDiffusionApplication/python_scripts/python_solvers_wrapper_convection_diffusion.py

```diff
@@ -56,9 +56,28 @@ def CreateSolverByParameters(model, solver_settings, parallelism):
 
     # Solvers for MPI parallelism
     elif (parallelism == "MPI"):
-        err_msg =  "The requested parallel type MPI is not yet available!\n"
-        raise Exception(err_msg)
-
+        # Transient solvers
```

Comment:
why do we need to add this block here? wouldn't it be better to have the test and check for the error block to be thrown just above?

Reply:
We need it because not all the transient solvers have MPI version available (the explicit and semi Eulerian parallelization is not as straightforward).

Reply:
well ok, i thought we could put above with a shorter code, but nevermind